packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-echarts/BoxPlot/data.ts

```diff
@@ -0,0 +1,122 @@
+export default [
```

Comment:
Just an aside/thought, but I know the E2E tests for most plugins are in Superset, not in the Plugins themselves. 

I'm hoping we can find a way to make the E2E tests live alongside the plugins themselves. On top of that, this data file makes me wonder if Jest/Enzyme tests that `mount` the viz from this Storybook entry, with all this test data, might be sufficient enough.

Reply:
True, I think whatever Storybook is doing should constitute an equivalent of an e2e test. I can look into how the two could somehow be combined.